index.bs

```diff
@@ -405,6 +407,8 @@ When this method is invoked on an {{XRFrame}} |frame|, the user agent MUST run t
 
 </div>
 
+NOTE: if any of the spaces belonging to the same {{XRHand}} return <code>null</null> when [=populate the pose|populating the pose=], all the spaces of that {{XRHand}} must also be marked as untracked.
```

Comment:
What does it mean to "mark a space as untracked" ?

Reply:
I agree that this makes it confusing. I will rephrase.